CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # dynsampler-go changelog
 
+## 0.3.0 2022-12-07
+
+### ⚠️ Breaking Changes ⚠️
+
+- Minimum Go version required is 1.16
```

Comment:
I think we agreed that eliminating out-of-support versions of languages is not a breaking change.

Reply:
We should write these things down! 😁 

The last time we dropped Go versions from libhoney-go, we opted to bump-minor and [note the breaking/might-surprise-you change in the changelog](https://github.com/honeycombio/libhoney-go/blob/main/CHANGELOG.md#1160-2022-07-13).

What do you think about dialing this subheading's wording down to "**⚠️ Attention ⚠️**"?

Reply:
We must've had opinions when we did libhoney-go, because we had to go back and sort out what we futzed up with versioning in Go. I'm a fan of following what we did there. Call out that it's breaking (because if someone is still on an older Go version then this is breaking), but no major bumps.

Reply:
Right! That was the lesson libhoney-go taught us: dropping old Go support is not bump-major-worthy in the Go ecosystem. If the library has moved to using Go language features not present in older Go, users† of the library still building with older Go will learn of the incompatibility during build at the latest. They can choose to pin to an older version of the library or upgrade their Go to one supported by the later version of the library.

With all that, I did not bump minor (0.2.1 -> 0.3.0). I'm not invested in calling it "breaking changes." I am suggesting we add some sort of emphasis on the drop support.
 
† "users of the library"? … 🤔 … Patrons!